### PR TITLE
🎨💄Consultancy hero section #176

### PR DESCRIPTION
**Pull request by PerisOduol618**
# Description

Consultancy hero section
Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers [here](https://github.com/italanta/elewa-website/issues/176).

Fixes # (issue) - *Example*: Fixes #167, NOTE: GitHub issue numbers should be prefixed with **#** to aid with issue tracking.

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## Screenshot (optional)
Screenshot from 2023-09-05 10-50-31
Screenshot from 2023-09-05 13-20-00

## Checklist

- [ X] When rendered the component is pixel perfect compared to the design above.
- [X ] The component is clean and does not have more than 30 lines
- [ X] The component SCSS uses CSS variables for colour, font, ... Connected to the global style.scss
- [X ] The design is responsive for screen, tablet and phones

### CODE REVIEW

File: libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            [buttonData]="buttonData"
+          ></elewa-website-slider-button>
+        </div>
+      </div>
+      <div class="bg-blocks">
+        <div class="block1"></div>
+        <div class="block2"></div>
+      </div>
+    </div> -->
+  <!-- </div> -->
```
Comment (AnnaL001) [inline libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.html:38]: To eliminate dead code, commented out code should be removed

File: libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+export class ConsultancyHeroSectionComponent {
+  heroSectionData = {
+    title: 'Learning design',
+    subtitle: '',
+    description:
+      'Your organisation has the potential to turn every task into an opportunity to learn. From growing crops, to nursing someone to good health, to developing the next-generation technology, let’s work together to make the magic happen.',
+  };
+  buttonData = {
+    text: 'Unlock your potential',
+  };
```
Comment (AnnaL001) [inline libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.ts:17]: To keep the component lean and ensure readability and maintainability, the 'heroSectionData' and 'buttonData' variable content can be moved to a Typescript file in the data module and referenced from there

File: libs/features/pages/consultancy-page/src/lib/consultancy-page.module.ts [hunk 3]
```diff
@@ -4,9 +4,12 @@ import { CommonModule } from '@angular/common';
 import { ConsultancyRoutingModule } from './consultancy.routing';
 
 import { ConsultancyPageComponent } from './main/consultancy-page/consultancy-page.component';
+import { ConsultancyHeroSectionComponent } from './components/consultancy-hero-section/consultancy-hero-section.component';
+import { TextsModule } from '@elewa-website/elements/layout/texts';
+import { ButtonsModule } from '@elewa-website/elements/layout/buttons';
```
Comment (AnnaL001) [inline libs/features/pages/consultancy-page/src/lib/consultancy-page.module.ts:10]: Kindly, observe the order of imports; the elements imports should appear before local imports.

**Review by AnnaL001 — CHANGES_REQUESTED** [review]
Good job, a few changes before this is merged.

Also for the hero section to match the specified design, the left margin can be reduced so that it matches that of the top navigation bar

**Comment by LemmyMwaura** [comment]
Closing this as it's been resolved

**Pull request closed without merging**